### PR TITLE
Add Dutch translation

### PR DESCRIPTION
**Pull request by Geoffry304**

### CODE REVIEW

**Comment by miraro3** [comment]
@mohitpanjwani I am familiar with Dutch indeed! This file looks complete. There seem to be a few typo's in there, but considering the size of the file I'd say this is a very good start.

**Comment by Geoffry304** [comment]
I did a quick check for typos, tell me if you find some more.

File: resources/assets/js/plugins/nl.json [hunk 1]
```diff
@@ -707,7 +707,7 @@
             "created_message": "Uitgavencategorie succevol aangemaakt",
             "deleted_message": "Uitgavencategorie succevol verwijderd",
```
Comment (ryanlupescu) [inline resources/assets/js/plugins/nl.json]:
```suggestion
            "deleted_message": "Uitgavencategorie succevol verwijderd",
```

File: resources/assets/js/plugins/nl.json [hunk 2]
```diff
@@ -707,7 +707,7 @@
             "created_message": "Uitgavencategorie succevol aangemaakt",
```
Comment (ryanlupescu) [inline resources/assets/js/plugins/nl.json]:
```suggestion
            "created_message": "Uitgavencategorie succesvol aangemaakt",
```

File: resources/assets/js/plugins/nl.json (hunk 1 above)
Comment (ryanlupescu) [inline resources/assets/js/plugins/nl.json]:
```suggestion
            "deleted_message": "Uitgavencategorie succesvol verwijderd",
```

**Comment by mohitpanjwani** [comment]
Thank you for the PR @Geoffry304 and @ryanlupescu @miraro3 for helping out. 

Please let me know when this is ready to merge :)

**Comment by Geoffry304** [comment]
@mohitpanjwani I fixed the typos so I think everything is good now for me :)

**Comment by Geoffry304** [comment]
Will this be merged ?

**Comment by mohitpanjwani** [comment]
Hey! Sorry I thought this PR is still pending. 

We will review and merge this tomorrow.

**Comment by mohitpanjwani** [comment]
```
"general": {
    "sort_by": "sorteer op",
    "ascending": "Oplopend",
    "descending": "Aflopend",
    "subject": "onderwerpen",
    "message": "Bericht",
    "test_mail_conf": "Test de e-mailconfiguratie",
    "send_mail_successfully": "Mail is succesvol verzonden",
}
"items": {
    "taxes": "Belastingen",
}
"invoices": {
    "cloned_successfully": "Factuur succesvol gekloond",
    "clone_invoice": "Factuur klonen",
    "confirm_clone": "Deze factuur wordt gekloond in een nieuwe factuur",
}
"payments": {
    "send_payment_receipt": "Betaalbewijs verzenden",
    "confirm_send_payment": "Deze betaling wordt via e-mail naar de klant gestuurd",
    "send_payment_successfully": "Betaling succesvol verzonden",
    "user_email_does_not_exist": "E-mailadres van gebruiker bestaat niet",
    "something_went_wrong": "Er is iets fout gegaan",
}
"expenses": {
    "select_a_customer": "Selecteer een klant",
    "customer": "Klanten",
}
"settings": {
    "customization": {
        "items": {
            "title": "Artikelen",
            "units": "eenheden",
            "add_item_unit": "Itemeenheid toevoegen",
            "unit_name": "Naam eenheid",
            "item_unit_added": "Item Unit toegevoegd",
            "item_unit_updated": "Artikeleenheid geüpdatet",
            "item_unit_confirm_delete": "U kunt dit item niet terughalen",
            "already_in_use": "Item Unit is al in gebruik",
            "deleted_message": "Artikeleenheid succesvol verwijderd"
        },
        "payments": {
            "payment_mode": "Betalingsmiddel",
            "add_payment_mode": "Betaalmodus toevoegen",
            "mode_name": "Mode Name",
            "payment_mode_added": "Betaalmethode toegevoegd",
            "payment_mode_updated": "Betalingsmodus geüpdatet",
            "payment_mode_confirm_delete": "U kunt deze betalingsmodus niet herstellen",
            "already_in_use": "De betalingsmodus is al in gebruik",
            "deleted_message": "Betaalmethode succesvol verwijderd"
        }
    }
    "validation": {
        "item_unit_already_taken": "De naam van dit item is al in gebruik",
        "payment_mode_already_taken": "Deze naam voor de betalingsmodus is al in gebruik",
        "subject_maxlength": "Het onderwerp mag niet meer dan 100 tekens bevatten.",
        "message_maxlength": "Bericht mag niet groter zijn dan 255 tekens.",    
        "something_went_wrong": "Er is iets fout gegaan"
    }
}
```

Hey @Geoffry304, I think you've missed the translations from the latest update so we haven't merged this yet. 

Is it possible for you to translate the above fields as well? 

Thanks

**Comment by weresort** [comment]
`"general": {
    "sort_by": "Sorteer op",
    "ascending": "Oplopend",
    "descending": "Aflopend",
    "subject": "Onderwerp",
    "message": "Bericht",
    "test_mail_conf": "Test de e-mailconfiguratie",
    "send_mail_successfully": "Mail is succesvol verzonden",
}
"items": {
    "taxes": "Belastingen",
}
"invoices": {
    "cloned_successfully": "Factuur succesvol gekopieerd",
    "clone_invoice": "Factuur kopiëren",
    "confirm_clone": "Deze factuur wordt gekopieerd naar een nieuwe factuur",
}
"payments": {
    "send_payment_receipt": "Betaalbewijs verzenden",
    "confirm_send_payment": "Dit betaalbewijs wordt via e-mail naar de klant gestuurd",
    "send_payment_successfully": "Betaling succesvol verzonden",
    "user_email_does_not_exist": "E-mailadres van gebruiker bestaat niet",
    "something_went_wrong": "Er is iets fout gegaan",
}
"expenses": {
    "select_a_customer": "Selecteer een klant",
    "customer": "Klant",
}
"settings": {
    "customization": {
        "items": {
            "title": "Artikelen",
            "units": "eenheden",
            "add_item_unit": "Artikel eenheid toevoegen",
            "unit_name": "Naam eenheid",
            "item_unit_added": "Artikel eenheid toegevoegd",
            "item_unit_updated": "Artikel eenheid geüpdatet",
            "item_unit_confirm_delete": "U kunt deze artikel eenheid niet terughalen",
            "already_in_use": "Artikel eenheid is al in gebruik",
            "deleted_message": "Artikel eenheid succesvol verwijderd"
        },
        "payments": {
            "payment_mode": "Betaalmethode",
            "add_payment_mode": "Betaalmethode toevoegen",
            "mode_name": "Mode naam",
            "payment_mode_added": "Betaalmethode toegevoegd",
            "payment_mode_updated": "Betaalmethode geüpdate",
            "payment_mode_confirm_delete": "U kunt deze betaalmethode niet herstellen",
            "already_in_use": "Deze betaalmethode is al in gebruik",
            "deleted_message": "Betaalmethode succesvol verwijderd"
        }
    }
    "validation": {
        "item_unit_already_taken": "De naam van dit artikel is al in gebruik",
        "payment_mode_already_taken": "Deze naam voor de betalingsmethode is al in gebruik",
        "subject_maxlength": "Het onderwerp mag niet meer dan 100 tekens bevatten.",
        "message_maxlength": "Bericht mag niet groter zijn dan 255 tekens.",    
        "something_went_wrong": "Er is iets fout gegaan"
    }
}`

**Comment by weresort** [comment]
mohitpanjwani 
i check above translation for you and make it all 100% good Dutch.

**Comment by mohitpanjwani** [comment]
Thanks a lot, @weresort and @Geoffry304.. 

but I cannot merge the PR as it has 2 minor conflicts.. @Geoffry304 do you have time to add the above translations provided by @weresort and also fix the conflicts? 

I'll merge it as soon as that's done. 

Thank you

**Comment by Geoffry304** [comment]
I try to check it today.

**Comment by mohitpanjwani** [comment]
@Geoffry304 this looks good thanks but there are still 2 conflicts so I cannot merge it yet.

**Comment by mohitpanjwani** [comment]
Hi @Geoffry304, do you have time to fix the conflicts this week? 

I was hoping to include the changes in this week's update.

**Comment by whitetornado** [comment]
any news regarding the Dutch translation??

**Comment by mohitpanjwani** [comment]
I think Geoffry is busy so we will try to make the changes ourselves and merge the PR.

**Comment by Loosie94** [comment]
I'm waiting for this as well, hope you guys merge soon :)

**Comment by mohitpanjwani** [comment]
Sorry for the wait guys. 

This will be merged and deployed in the next week’s minor update.

**Comment by Loosie94** [comment]
@mohitpanjwani Spent some time today to translate Dutch with the latest Crater version. I've seen some strings we're missing so I've added them. See my pull request here: https://github.com/bytefury/crater/pull/232

**Pull request closed without merging**